packages/snaps-controllers/src/snaps/SnapController.test.ts

```diff
@@ -1109,14 +1058,16 @@ describe('SnapController', () => {
           id: 1,
         },
       }),
-    ).toBe(21896430);
+    ).toBe(21896426);
```

Comment:
How is this returning a number? 🤔

Reply:
I don't know. It was doing it before too though.